internal/build/docker_test.go

```diff
@@ -35,26 +36,6 @@ func TestDigestFromSingleStepOutput(t *testing.T) {
 	}
 }
 
-func TestBuildBase(t *testing.T) {
```

Comment:
We don't really need this test any more, it's entirely covered by the other test.